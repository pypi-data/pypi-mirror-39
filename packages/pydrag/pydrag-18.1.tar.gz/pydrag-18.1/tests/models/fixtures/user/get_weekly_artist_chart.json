{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&format=json&method=user.getWeeklyArtistChart&user=rj"
            },
            "response": {
                "body": {
                    "string": "{\"weeklyartistchart\":{\"artist\":[{\"name\":\"Womb Sounds Heartbeat\",\"mbid\":\"\",\"playcount\":\"35\",\"url\":\"https://www.last.fm/music/Womb+Sounds+Heartbeat\",\"@attr\":{\"rank\":\"1\"}},{\"name\":\"The Gaslight Anthem\",\"mbid\":\"CENSORED17e3\",\"playcount\":\"10\",\"url\":\"https://www.last.fm/music/The+Gaslight+Anthem\",\"@attr\":{\"rank\":\"2\"}},{\"name\":\"Kacey Musgraves\",\"mbid\":\"CENSORED40cb\",\"playcount\":\"3\",\"url\":\"https://www.last.fm/music/Kacey+Musgraves\",\"@attr\":{\"rank\":\"3\"}},{\"name\":\"Ferris & Sylvester\",\"mbid\":\"\",\"playcount\":\"2\",\"url\":\"https://www.last.fm/music/Ferris+&+Sylvester\",\"@attr\":{\"rank\":\"4\"}},{\"name\":\"Mario\",\"mbid\":\"CENSORED2cc7\",\"playcount\":\"2\",\"url\":\"https://www.last.fm/music/Mario\",\"@attr\":{\"rank\":\"5\"}},{\"name\":\"Nao\",\"mbid\":\"CENSORED1fed\",\"playcount\":\"2\",\"url\":\"https://www.last.fm/music/Nao\",\"@attr\":{\"rank\":\"6\"}},{\"name\":\"Amber Mark\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Amber+Mark\",\"@attr\":{\"rank\":\"7\"}},{\"name\":\"Ar'mon & Trey\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Ar%27mon+&+Trey\",\"@attr\":{\"rank\":\"8\"}},{\"name\":\"Aretha Franklin\",\"mbid\":\"CENSORED299e\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Aretha+Franklin\",\"@attr\":{\"rank\":\"9\"}},{\"name\":\"Brian Deady\",\"mbid\":\"CENSORED1726\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Brian+Deady\",\"@attr\":{\"rank\":\"10\"}},{\"name\":\"Catherine McGrath\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Catherine+McGrath\",\"@attr\":{\"rank\":\"11\"}},{\"name\":\"Childish Gambino\",\"mbid\":\"CENSORED607e\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Childish+Gambino\",\"@attr\":{\"rank\":\"12\"}},{\"name\":\"Dram\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Dram\",\"@attr\":{\"rank\":\"13\"}},{\"name\":\"Enno Aare\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Enno+Aare\",\"@attr\":{\"rank\":\"14\"}},{\"name\":\"Filmore\",\"mbid\":\"CENSORED0c76\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Filmore\",\"@attr\":{\"rank\":\"15\"}},{\"name\":\"Fleetwood Mac\",\"mbid\":\"CENSORED5b1a\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Fleetwood+Mac\",\"@attr\":{\"rank\":\"16\"}},{\"name\":\"Franz Schubert\",\"mbid\":\"CENSORED65ed\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Franz+Schubert\",\"@attr\":{\"rank\":\"17\"}},{\"name\":\"Ilham\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Ilham\",\"@attr\":{\"rank\":\"18\"}},{\"name\":\"India.Arie\",\"mbid\":\"CENSOREDefdd\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/India.Arie\",\"@attr\":{\"rank\":\"19\"}},{\"name\":\"Into The Ark\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Into+The+Ark\",\"@attr\":{\"rank\":\"20\"}},{\"name\":\"Jackie Wilson\",\"mbid\":\"CENSORED05dd\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Jackie+Wilson\",\"@attr\":{\"rank\":\"21\"}},{\"name\":\"JamesDavis\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/JamesDavis\",\"@attr\":{\"rank\":\"22\"}},{\"name\":\"Kiana Led\u00e9\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Kiana+Led%C3%A9\",\"@attr\":{\"rank\":\"23\"}},{\"name\":\"Kylie Minogue\",\"mbid\":\"CENSORED684c\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Kylie+Minogue\",\"@attr\":{\"rank\":\"24\"}},{\"name\":\"Liza\",\"mbid\":\"CENSORED8446\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Liza\",\"@attr\":{\"rank\":\"25\"}},{\"name\":\"Maddie & Tae\",\"mbid\":\"CENSOREDb79f\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Maddie+&+Tae\",\"@attr\":{\"rank\":\"26\"}},{\"name\":\"Major.\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Major.\",\"@attr\":{\"rank\":\"27\"}},{\"name\":\"Marsha Ambrosius\",\"mbid\":\"CENSOREDd8aa\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Marsha+Ambrosius\",\"@attr\":{\"rank\":\"28\"}},{\"name\":\"masego\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/masego\",\"@attr\":{\"rank\":\"29\"}},{\"name\":\"Melat\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Melat\",\"@attr\":{\"rank\":\"30\"}},{\"name\":\"Nikki Blonsky\",\"mbid\":\"CENSOREDfd5d\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Nikki+Blonsky\",\"@attr\":{\"rank\":\"31\"}},{\"name\":\"Njomza\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Njomza\",\"@attr\":{\"rank\":\"32\"}},{\"name\":\"Paul Simon\",\"mbid\":\"CENSOREDebb9\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Paul+Simon\",\"@attr\":{\"rank\":\"33\"}},{\"name\":\"Queen Naija\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Queen+Naija\",\"@attr\":{\"rank\":\"34\"}},{\"name\":\"R.Lum.R\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/R.Lum.R\",\"@attr\":{\"rank\":\"35\"}},{\"name\":\"Ro James\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Ro+James\",\"@attr\":{\"rank\":\"36\"}},{\"name\":\"Ryan Trey\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Ryan+Trey\",\"@attr\":{\"rank\":\"37\"}},{\"name\":\"Sat-B\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Sat-B\",\"@attr\":{\"rank\":\"38\"}},{\"name\":\"Sauti Sol\",\"mbid\":\"CENSORED80b9\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Sauti+Sol\",\"@attr\":{\"rank\":\"39\"}},{\"name\":\"Shawn Stockman\",\"mbid\":\"CENSORED87d4\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Shawn+Stockman\",\"@attr\":{\"rank\":\"40\"}},{\"name\":\"Stevie Wonder\",\"mbid\":\"CENSOREDe986\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Stevie+Wonder\",\"@attr\":{\"rank\":\"41\"}},{\"name\":\"Summer Walker\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Summer+Walker\",\"@attr\":{\"rank\":\"42\"}},{\"name\":\"Tamia\",\"mbid\":\"CENSORED7396\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Tamia\",\"@attr\":{\"rank\":\"43\"}},{\"name\":\"The Wandering Hearts\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/The+Wandering+Hearts\",\"@attr\":{\"rank\":\"44\"}},{\"name\":\"The Wild Things\",\"mbid\":\"CENSOREDae52\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/The+Wild+Things\",\"@attr\":{\"rank\":\"45\"}},{\"name\":\"Van Morrison\",\"mbid\":\"CENSORED3559\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Van+Morrison\",\"@attr\":{\"rank\":\"46\"}},{\"name\":\"Free Nationals\",\"mbid\":\"\",\"playcount\":\"1\",\"url\":\"https://www.last.fm/music/Free+Nationals\",\"@attr\":{\"rank\":\"47\"}}],\"@attr\":{\"user\":\"RJ\",\"from\":\"1541160000\",\"to\":\"1541764800\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "6456"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:28:22 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
