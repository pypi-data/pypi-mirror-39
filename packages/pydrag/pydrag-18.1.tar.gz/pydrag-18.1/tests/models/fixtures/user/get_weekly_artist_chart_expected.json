{
    "data": [
        {
            "mbid": "",
            "name": "Womb Sounds Heartbeat",
            "playcount": 35,
            "rank": 1,
            "url": "https://www.last.fm/music/Womb+Sounds+Heartbeat"
        },
        {
            "mbid": "CENSORED17e3",
            "name": "The Gaslight Anthem",
            "playcount": 10,
            "rank": 2,
            "url": "https://www.last.fm/music/The+Gaslight+Anthem"
        },
        {
            "mbid": "CENSORED40cb",
            "name": "Kacey Musgraves",
            "playcount": 3,
            "rank": 3,
            "url": "https://www.last.fm/music/Kacey+Musgraves"
        },
        {
            "mbid": "",
            "name": "Ferris & Sylvester",
            "playcount": 2,
            "rank": 4,
            "url": "https://www.last.fm/music/Ferris+&+Sylvester"
        },
        {
            "mbid": "CENSORED2cc7",
            "name": "Mario",
            "playcount": 2,
            "rank": 5,
            "url": "https://www.last.fm/music/Mario"
        },
        {
            "mbid": "CENSORED1fed",
            "name": "Nao",
            "playcount": 2,
            "rank": 6,
            "url": "https://www.last.fm/music/Nao"
        },
        {
            "mbid": "",
            "name": "Amber Mark",
            "playcount": 1,
            "rank": 7,
            "url": "https://www.last.fm/music/Amber+Mark"
        },
        {
            "mbid": "",
            "name": "Ar'mon & Trey",
            "playcount": 1,
            "rank": 8,
            "url": "https://www.last.fm/music/Ar%27mon+&+Trey"
        },
        {
            "mbid": "CENSORED299e",
            "name": "Aretha Franklin",
            "playcount": 1,
            "rank": 9,
            "url": "https://www.last.fm/music/Aretha+Franklin"
        },
        {
            "mbid": "CENSORED1726",
            "name": "Brian Deady",
            "playcount": 1,
            "rank": 10,
            "url": "https://www.last.fm/music/Brian+Deady"
        },
        {
            "mbid": "",
            "name": "Catherine McGrath",
            "playcount": 1,
            "rank": 11,
            "url": "https://www.last.fm/music/Catherine+McGrath"
        },
        {
            "mbid": "CENSORED607e",
            "name": "Childish Gambino",
            "playcount": 1,
            "rank": 12,
            "url": "https://www.last.fm/music/Childish+Gambino"
        },
        {
            "mbid": "",
            "name": "Dram",
            "playcount": 1,
            "rank": 13,
            "url": "https://www.last.fm/music/Dram"
        },
        {
            "mbid": "",
            "name": "Enno Aare",
            "playcount": 1,
            "rank": 14,
            "url": "https://www.last.fm/music/Enno+Aare"
        },
        {
            "mbid": "CENSORED0c76",
            "name": "Filmore",
            "playcount": 1,
            "rank": 15,
            "url": "https://www.last.fm/music/Filmore"
        },
        {
            "mbid": "CENSORED5b1a",
            "name": "Fleetwood Mac",
            "playcount": 1,
            "rank": 16,
            "url": "https://www.last.fm/music/Fleetwood+Mac"
        },
        {
            "mbid": "CENSORED65ed",
            "name": "Franz Schubert",
            "playcount": 1,
            "rank": 17,
            "url": "https://www.last.fm/music/Franz+Schubert"
        },
        {
            "mbid": "",
            "name": "Ilham",
            "playcount": 1,
            "rank": 18,
            "url": "https://www.last.fm/music/Ilham"
        },
        {
            "mbid": "CENSOREDefdd",
            "name": "India.Arie",
            "playcount": 1,
            "rank": 19,
            "url": "https://www.last.fm/music/India.Arie"
        },
        {
            "mbid": "",
            "name": "Into The Ark",
            "playcount": 1,
            "rank": 20,
            "url": "https://www.last.fm/music/Into+The+Ark"
        },
        {
            "mbid": "CENSORED05dd",
            "name": "Jackie Wilson",
            "playcount": 1,
            "rank": 21,
            "url": "https://www.last.fm/music/Jackie+Wilson"
        },
        {
            "mbid": "",
            "name": "JamesDavis",
            "playcount": 1,
            "rank": 22,
            "url": "https://www.last.fm/music/JamesDavis"
        },
        {
            "mbid": "",
            "name": "Kiana Led\u00e9",
            "playcount": 1,
            "rank": 23,
            "url": "https://www.last.fm/music/Kiana+Led%C3%A9"
        },
        {
            "mbid": "CENSORED684c",
            "name": "Kylie Minogue",
            "playcount": 1,
            "rank": 24,
            "url": "https://www.last.fm/music/Kylie+Minogue"
        },
        {
            "mbid": "CENSORED8446",
            "name": "Liza",
            "playcount": 1,
            "rank": 25,
            "url": "https://www.last.fm/music/Liza"
        },
        {
            "mbid": "CENSOREDb79f",
            "name": "Maddie & Tae",
            "playcount": 1,
            "rank": 26,
            "url": "https://www.last.fm/music/Maddie+&+Tae"
        },
        {
            "mbid": "",
            "name": "Major.",
            "playcount": 1,
            "rank": 27,
            "url": "https://www.last.fm/music/Major."
        },
        {
            "mbid": "CENSOREDd8aa",
            "name": "Marsha Ambrosius",
            "playcount": 1,
            "rank": 28,
            "url": "https://www.last.fm/music/Marsha+Ambrosius"
        },
        {
            "mbid": "",
            "name": "masego",
            "playcount": 1,
            "rank": 29,
            "url": "https://www.last.fm/music/masego"
        },
        {
            "mbid": "",
            "name": "Melat",
            "playcount": 1,
            "rank": 30,
            "url": "https://www.last.fm/music/Melat"
        },
        {
            "mbid": "CENSOREDfd5d",
            "name": "Nikki Blonsky",
            "playcount": 1,
            "rank": 31,
            "url": "https://www.last.fm/music/Nikki+Blonsky"
        },
        {
            "mbid": "",
            "name": "Njomza",
            "playcount": 1,
            "rank": 32,
            "url": "https://www.last.fm/music/Njomza"
        },
        {
            "mbid": "CENSOREDebb9",
            "name": "Paul Simon",
            "playcount": 1,
            "rank": 33,
            "url": "https://www.last.fm/music/Paul+Simon"
        },
        {
            "mbid": "",
            "name": "Queen Naija",
            "playcount": 1,
            "rank": 34,
            "url": "https://www.last.fm/music/Queen+Naija"
        },
        {
            "mbid": "",
            "name": "R.Lum.R",
            "playcount": 1,
            "rank": 35,
            "url": "https://www.last.fm/music/R.Lum.R"
        },
        {
            "mbid": "",
            "name": "Ro James",
            "playcount": 1,
            "rank": 36,
            "url": "https://www.last.fm/music/Ro+James"
        },
        {
            "mbid": "",
            "name": "Ryan Trey",
            "playcount": 1,
            "rank": 37,
            "url": "https://www.last.fm/music/Ryan+Trey"
        },
        {
            "mbid": "",
            "name": "Sat-B",
            "playcount": 1,
            "rank": 38,
            "url": "https://www.last.fm/music/Sat-B"
        },
        {
            "mbid": "CENSORED80b9",
            "name": "Sauti Sol",
            "playcount": 1,
            "rank": 39,
            "url": "https://www.last.fm/music/Sauti+Sol"
        },
        {
            "mbid": "CENSORED87d4",
            "name": "Shawn Stockman",
            "playcount": 1,
            "rank": 40,
            "url": "https://www.last.fm/music/Shawn+Stockman"
        },
        {
            "mbid": "CENSOREDe986",
            "name": "Stevie Wonder",
            "playcount": 1,
            "rank": 41,
            "url": "https://www.last.fm/music/Stevie+Wonder"
        },
        {
            "mbid": "",
            "name": "Summer Walker",
            "playcount": 1,
            "rank": 42,
            "url": "https://www.last.fm/music/Summer+Walker"
        },
        {
            "mbid": "CENSORED7396",
            "name": "Tamia",
            "playcount": 1,
            "rank": 43,
            "url": "https://www.last.fm/music/Tamia"
        },
        {
            "mbid": "",
            "name": "The Wandering Hearts",
            "playcount": 1,
            "rank": 44,
            "url": "https://www.last.fm/music/The+Wandering+Hearts"
        },
        {
            "mbid": "CENSOREDae52",
            "name": "The Wild Things",
            "playcount": 1,
            "rank": 45,
            "url": "https://www.last.fm/music/The+Wild+Things"
        },
        {
            "mbid": "CENSORED3559",
            "name": "Van Morrison",
            "playcount": 1,
            "rank": 46,
            "url": "https://www.last.fm/music/Van+Morrison"
        },
        {
            "mbid": "",
            "name": "Free Nationals",
            "playcount": 1,
            "rank": 47,
            "url": "https://www.last.fm/music/Free+Nationals"
        }
    ],
    "from_date": 1541160000,
    "to_date": 1541764800,
    "user": "RJ"
}
